recipe/meta.yaml

```diff
@@ -75,3 +55,10 @@ about:
 extra:
   recipe-maintainers:
     - scopatz
+    - gonuke
+    - bam241
+    - kkiesling
+    - a-co
+    - opotowsky
+    - katyhuff
+    - pshriwise
```

Comment:
@gonuke @bam241 @a-co  @opotowsky @katyhuff @pshriwise - are you OK with being maintainers?

Reply:
I am OK to be a maintainer

Reply:
I'm ok with being a maintainer as well (in case the emoji isn't good enough)

Reply:
I'm OK with being a maintainer